watertap/flowsheets/electroNP/BSM2_electroNP_no_bioP.py

```diff
@@ -627,6 +627,8 @@ def solve(m, solver=None):
     if m.fs.has_electroNP is False:
         stream_table = create_stream_table_dataframe(
             {
+                "R3 inlet": m.fs.R3.inlet,
+                "ASM-ADM translator inlet": m.fs.translator_asm2d_adm1.inlet,
                 "Feed": m.fs.FeedWater.outlet,
```

Comment:
You can delete these two outputs for R3 inlet and ASM-ADM translator inlet, I used them for getting tear_guess but don't need to be shown in the report.